docs/GITOPS-CONFIG.md

```diff
@@ -111,9 +121,9 @@ You will also need to create a Secret in your chosen namespace with the followin
 4. `ARGOCD_API_TOKEN`
 
 #### Step 3: Updating Plugins
-You will follow the same steps as [step 2 for the ai-rhdh-installer](#ai-rhdh-installer-script-configuration)
+You will follow the same steps as [step 2 for the ai-rhdh-installer](#step-2-updating-plugins).
 
 #### Step 4: Updating RHDH Deployment
-Once you have applied the ConfigMaps and Secrets to your cluster and the necessary namespace you can now follow the same steps in [step 3 for the ai-rhdh-installer](#ai-rhdh-installer-manual-configuration). Every step will be identical if your RHDH instance was created using the `Red Hat Developer Hub Operator`, however, if you installed RHDH using the `Helm Chart` you may find that `.spec.template.spec.containers.envFrom` does not exist in the Deployment yaml. If this is the case you can simply add that field and the necessary secret information.
+Once you have applied the ConfigMaps and Secrets to your cluster and the necessary namespace you can now follow the same steps in [step 3 for the ai-rhdh-installer](#step-3-updating-rhdh-deployment). Every step will be identical if your RHDH instance was created using the `Red Hat Developer Hub Operator`, however, if you installed RHDH using the `Helm Chart` you may find that `.spec.template.spec.containers.envFrom` does not exist in the Deployment yaml. If this is the case you can simply add that field and the necessary secret information.
```

Comment:
```suggestion
Once you have applied the ConfigMaps and Secrets to your cluster and the chosen namespace you can now follow the same steps in [step 3 for the ai-rhdh-installer](#step-3-updating-rhdh-deployment). Every step will be identical if your RHDH instance was created using the `Red Hat Developer Hub Operator`, however, if you installed RHDH using the `Helm Chart` you may find that `.spec.template.spec.containers.envFrom` does not exist in the Deployment yaml. If this is the case you can simply add that field and the necessary secret information.
```

Reply:
> Might be better also here to be more detailed:
> 
>     * `If this is the case you can simply add that field and the necessary secret information`
> 
> 
> I think maybe add this info as troubleshooting information might be a clearer approach? WDYT?

@thepetk Yeah I agree, I think specifying `envFrom` again here would make it more clear, since there is a lot in this paragraph it made me think which field it was referring to as well. Also rewording to 'secret reference' rather than 'secret information'.

Reply:
Done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/b2898c501fd7b26a578ec9cdd15f7da7c58655bf